### DISCUSSION TITLE
Please delete this Glossary term

### DISCUSSION THREAD

**Discussion by baffclan** [WeblateOrg/weblate#5822] [General] [answered]
Please delete this Glossary term.
https://hosted.weblate.org/translate/weblate/glossary/ja/?checksum=ec0511231b71f637
There is an unnecessary description in Key.

**Comment by rffontenelle** [comment, marked as answer]
@baffclan One can remove glossary terms in _Tools > Delete string_ in this very same screen. See [documentation](https://docs.weblate.org/en/latest/user/glossary.html#glossary-terms) for this menu item

**Reply by baffclan** [reply]
@rffontenelle Thanks for your reply.
I also tried Tools> Delete string.
But I couldn't delete it.

**Reply by rffontenelle** [reply]
Done. I too was unable to remove the string you mentioned above, but I was able to delete the other string with same translation (インタプリタ), which resulted in deleting both (go figure!).

By the way, the one I deleted had your comment:
> interpreter は三か所で使われており、ひとつは未翻訳で、インタープリター と インタプリター の揺れがありました。とりあえず Wikipwdia の用語 インタプリタ を入れておきます。

**Reply by baffclan** [reply]
Thank you for deleting it!
It just seems like a problem has occurred. 
https://github.com/WeblateOrg/glossary/commit/e4ed4c24f4788e9b7a31e4ddf0a819fa287023f6?branch=e4ed4c24f4788e9b7a31e4ddf0a819fa287023f6&diff=split
![Screenshot SSO term](https://user-images.githubusercontent.com/18899730/114328991-38218100-9b79-11eb-9369-22e376206396.png)

**Reply by rffontenelle** [reply]
I believe `<term/>` is not an issue. Look at this explanation I found in https://stackoverflow.com/questions/28512655/what-is-the-main-difference-of-br-and-br/28512738:
> ... the difference is the position, and is universal for all XML tags. <TAG_NAME> indicates the beginning of a tag, and </TAG_NAME> indicates the end of a tag. When a tag is used with nothing between it, then a self-closing, or null tag can be used, which combines the beginning and end. It looks like <TAG_NAME />.

So `<term></term>` is the same as `<term/>`, which might have appeared after some change in the code (just guessing here).

**Reply by baffclan** [reply]
Oops! I filed this problem to issue https://github.com/WeblateOrg/weblate/issues/5827 .
